.codebuild/run_ios_modelgen_e2e_test.yml

```diff
@@ -10,7 +10,12 @@ env:
 phases:
   build:
     commands:
+      - source ./shared-scripts.sh && _setupE2ETestsLinux
+      - codebuild-breakpoint
       - source ./shared-scripts.sh && _runE2ETestsLinux
+      - unset AWS_ACCESS_KEY_ID
+      - unset AWS_SECRET_ACCESS_KEY
+      - unset AWS_SESSION_TOKEN
```

Comment:
We need to unset these to fall back to codebuild role from the E2E parent role before we populate the generated models in S3 artifacts bucket.